src/System.Runtime/tests/System/Lazy.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace System.Runtime.Tests
 {
-    public static class LazyInitializerTests
```

Comment:
No, I wasn't planning on it.  I don't think we should be in the business of manually copying and maintaining two separate copies of tests.  And if we must, then we should do so methodically rather than for one-offs like this; there have been many tests added on GitHub that, to my knowledge, have not been manually migrated to another system.